Prolangle/Pages/Home.razor

```diff
@@ -23,17 +23,42 @@
 else
 {
 	<LanguageSelection AvailableLanguages="@AvailableLanguages" OnLanguageSelected="@OnLanguageSelected"/>
+
+	<p>
+		<Icon Name="IconName.Lightbulb" Padding="Padding.Is2" />
+		@if (guessCount == 0)
+		{
+			<span>
+				Lost? Just try picking a language, or head to <NavLink href="/explanations">Explanations</NavLink>
+				for more info.
+			</span>
+		}
+		else
+		{
+			<span>
+				Some terms are
+				<span style="text-decoration: underline; text-decoration-style: dotted; text-underline-offset: 2px; text-decoration-thickness: 1px;">underlined</span>.
```

Comment:
Maybe we can move the style for explainable tags to the global css file?

Reply:
I mean to do that and forgot, I guess.